src/utils/common_spaces.jl

```diff
@@ -74,6 +74,38 @@ function make_horizontal_space(mesh, quad, comms_ctx, bubble)
     return space
 end
 
+function diffuse_surface_elevation_biharmonic!(
```

Comment:
Can you add docstring to this function, including references to where the equations came from?

Reply:
Sure; I can add better docstrings; these functions are implementations of unsteady diffusion equations applied to the surface orography.

Reply:
@Sbozzolo I'll move this function directly to ClimaCore (within the Hypsography module)